runtime/queries/agda/highlights.scm

```diff
@@ -1,38 +1,47 @@
-
 ;; Constants
 (integer) @number
 
 ;; Variables and Symbols
```

Comment:
Question: do we want to standardize single vs. double semicolon for comments? (I'd say yes, especially if we leave inline comments.)

Reply:
I don't know. Whichever you prefer will be what I follow.

Reply:
Depends on how simple it is to implement ;)

Reply:
I see it as a `(#gsub! @capture "^;+(.*)" ";;%1")`, then modifying a bit to also apply metadata for get_node_text

Reply:
That sounds feasible. The two options are

1. Top-level comments get two `;;`; inline comments get (properly) moved and `;`.
2. All comments get one `;`;  inline comments stay where they are.

My (swayable) preference would be for option 2.

Reply:
gsub to the rescue (soon) then. I'll go with 2

Reply:
(We can always revisit later; the nice part about a full formatter is that it always yields the intended format, no matter the original state.)

Reply:
Well from a syntax tree POV, both nodes are adjacent and nothing else. Lines aren't really a thing for non-indent-aware languages. Still, I'll try to go for 2

Reply:
> Can't we distinguish with `#not-has-parent?`

We can. But as you said you're more in favor of 2., it won't be needed

Reply:
I celebrated too soon. `#gsub!` doesn't seem to be handling stuffs quite well. On the other hand, end of line comments are now preserved @clason

Reply:
It works now!